docs/docs/cloud/developer.mdx

```diff
@@ -0,0 +1,96 @@
+---
+id: developer
+title: Ory Cloud Developer Plan
+sidebar_label: Developer Plan
+---
+
+This document describes what you can expect when signing up for a Developer Plan
+in Ory Cloud.
+
+The Developer Plan includes one Ory Cloud Project for testing, research and
+personal projects.
+
+**Sign up at [console.ory.sh](https://console.ory.sh/registration).**
+
+### What you get
+
+- Free of charge Ory Cloud Project for testing, research and personal projects.
+
+Features:
+
+- Login
+- Registration
+- APIs for Web App
+- APIs for Single Page App
+- APIs for Mobile App
+- Secure Account Recovery
+- E-Mail Verification
+- Breached Password Detection
+- Bring your own UI
+- User Management
+- Custom Identity Models
+- Standard 2FA
+- FIDO2-compliant 2FA
+- Instant Log Out
+- Encryption in Transit
+- Administrate Identities, Cookies & Sessions
+- Pay for what you use in production (see Limits)
+
+Support options:
+
+- Dedicated Developer Relations
+- Chat support via public Slack channel
+- Ticket via GitHub Issues
+
+Have a look [at the roadmap](./cloud-roadmap.mdx).
+
+### Limits of Developer Plan
```

Comment:
das ist etwas negative dargestellt, hier würde ich schreiben "Included in the Developer Plan"